__init__.py

```diff
@@ -24,11 +24,26 @@ def __init__(self, config):
             _LOGGER.error("Missing auth token! You must set 'github-token' in your config")
         self.name = self.config.get("name", "github")
         self.opsdroid = None
-
+    @staticmethod
+    def get_site(url):
+        #print(url)
+        response = await aiohttp.request('GET', url)
+        print("Reading bot information...")
+        return (await response.read())
+    
     async def connect(self, opsdroid):
         """Connect to GitHub."""
         self.opsdroid = opsdroid
-
+        # had troubles using the python 3.6 syntax, using asyncio as the package
+        loop = self.opsdroid.eventloop
+        raw_json = loop.run_until_complete(
```

Comment:
One you make `get_site` a coroutine you do not need to mess around with the loop. You can simply `await self.get_site('https://api.github.com/user?access_token=' + self.github_token`.

Also `GITHUB_API_URL` is defined at the top, you might want to make use of it.